app/models/solr_document.rb

```diff
@@ -91,6 +91,22 @@ def title
     self[Solrizer.solr_name('title', :stored_searchable)] || nested_ordered_title_label || []
   end
 
+  def description
```

Comment:
We could extend this method to return existing unordered values if available as a fallback when the ordered values are not available yet. I think this would help during migration so that we can still display those values.

For example, we can do something like:
```
def description
     nested_ordered_description_label.present? ? nested_ordered_description_label : self[Solrizer.solr_name('description', :stored_searchable)]
  end
```